### DISCUSSION TITLE
Use cursor css class 'select-all' in documentation on primeflex css classes. Optional copy to clipboard on click?

### DISCUSSION THREAD

**Discussion by djmj** [primefaces/community#2312] [PrimeFlex]
Please use cursor 'select-all' on your css classes in documentation.

This makes it easier to copy it for development.

Optional add a button or onclick copy it to clipboard?

**Example**

https://primeflex.org/userselect

We manually have to select 'select-none' in the documentation. with cursor. Especially if there is a hyphen inside its more work then a doubleclick to select all.

Press copy on keyboard to get it into clipboard.

**Comment by melloware** [comment]
I think this got added here: https://github.com/primefaces/primeflex/issues/166

But the new showcase was not deployed.
